#!/usr/bin/env python
PACKAGE = "robot_vision"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("limit_area_r", int_t, 0, "Radius added value", -5, -15, 15)
gen.add("threshold_min", int_t, 0, "Threshold min", 100, 0, 255)
gen.add("threshold_max", int_t, 0, "Threshold max", 255, 0, 255)

exit(gen.generate(PACKAGE, "robot_vision", "limit_hsv"))
